app/views/trucks/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>All Trucks</h1>
+
+<% if @trucks.length > 0 %>
```

Comment:
Idiomatically, I would have written this `unless @trucks.emty?`. But that's just me.